### ISSUE TITLE
Claims array support (Audience, etc)

### ISSUE THREAD

**Issue by macarlet** [jwtk/jjwt#336]
Hi, reading the rfc about audience claim https://tools.ietf.org/html/rfc7519#section-4.1.3, it seems that the aud can be a list of values, but the api implement this claim just as a single value.

**Comment by lhazlewood** [comment]
Yep, you're right - thanks for the issue.  We have to do this for a similar set of other claims as well.

Can you use the `JwtBuilder#addClaims` method to work around this temporarily?

**Comment by macarlet** [comment]
Sure! Thanks!

**Comment by workmanw** [comment]
@lhazlewood Just out of curiosity is this something you would expect a PR for? Or would you rather just keep the audience simple without an array?

**Comment by lhazlewood** [comment]
Some thoughtful design needs to go into this and we just don't want to hold up 0.11 any longer - adding to a future release.

**Comment by martinth** [comment]
In the meantime, you can do this to get the audiences:

```java
Set<String> audiences = Set.copyOf(jwtToken.getBody().get("aud", ArrayList.class));
```

**Comment by Rambalac** [comment]
Well, you cannot. Java Maps do not allow you to convert into collections. And Java Maps do not keep duplicated keys.

**Comment by lhazlewood** [comment]
I realized we have a couple issues open relating to `aud` being an array.  Closing this as a duplicate of #77 which we hope to have fixed in the 0.12.0 release
